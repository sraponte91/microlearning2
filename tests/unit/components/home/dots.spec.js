import { mount } from '@vue/test-utils'
import dots from '@/components/home/dots.vue'

const wrapper = mount(dots)


describe('dots.vue', () => {
  test('Is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })
  test('Renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
})
