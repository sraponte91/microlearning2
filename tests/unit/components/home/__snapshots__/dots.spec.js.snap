// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dots.vue Renders correctly 1`] = `
<div
  class="one-dot d-flex position-absolute align-items-start d-none-991i"
>
  <div
    class="dots-div d-flex flex-wrap"
  >
    <div
      class="d-flex flex-wrap"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
  </div>
   
  <div
    class="dots-div only-for-dots d-flex flex-wrap"
  >
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
     
    <div
      class="d-flex"
    >
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
       
      <span
        class="dot"
      />
    </div>
  </div>
</div>
`;
